lib/music_album.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'item'
+
+class MusicAlbum < Item
```

Comment:
s per the UML class diagram, while creating a music album, it should ask for `Genre`, `Author`, and `Label` details. However, in your case, it is only asking for the Genre. Kindly fix this issue.